rpms/build.sh

```diff
@@ -51,11 +58,13 @@ $linux rpmbuild --define "_topdir ${rpm_build_dir}" --define "_threads ${threads
         --define "_debugenabled ${debug}" --target ${architecture_target} \
         -ba ${rpm_build_dir}/SPECS/${package_name}.spec
 
-rpm_file="wazuh-${build_target}-${wazuh_version}-${package_release}.${architecture_target}.rpm"
-pkg_path="${rpm_build_dir}/RPMS/${architecture_target}"
-
 if [[ "${checksum}" == "yes" ]]; then
     cd ${pkg_path} && sha512sum ${rpm_file} > /var/local/checksum/${rpm_file}.sha512
+    cd ${src_path} && sha512sum ${src_file} > /var/local/checksum/${src_file}.sha512
```

Comment:
This line will compute the checksum of the src package always. The script should only compute the checksum of that file only if the `--src` parameter is set.

Reply:
Fixed in d20150a.